src/styles/main.css

```diff
@@ -6,10 +6,25 @@
 
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700&family=Spartan:wght@400;500;600;700&display=swap');
 
+/* Scrollbar */
+::-webkit-scrollbar {
```

Comment:
have you tested this against Firefox? This may break since this seems to be a vendor-specific prop

Reply:
Since firefox is really limited in customizing the scrollbar, as for now I've changed it to a solid color (since gradient doesn't work) and the width to `thin`